File: 백준/Bronze/34543. 와우산 스탬프 투어/와우산 스탬프 투어.py
import sys
input=lambda:sys.stdin.readline().rstrip()
n,w=int(input()),int(input())
rlt=n*10
if n==5:
    rlt+=70
elif n>=3:
    rlt+=20
if w>1000:
    rlt = 0 if rlt-15<0 else rlt-15
print(rlt)

File: 백준/Bronze/34543. 와우산 스탬프 투어/와우산 스탬프 투어.c
#include <stdio.h>
int main(void) {
    int n,w,rlt;
    scanf("%d %d",&n,&w);
    rlt=n*10;
    if(n==5)
        rlt+=70;
    else if(n>=3)
        rlt+=20;
    if(w>1000)
        rlt=rlt-15<0?0:rlt-15;
    printf("%d",rlt);
    return 0;
}